frameworks/agno/travel-planner/agentuity_agents/travel_planner/query_parser.py

```
@@ -0,0 +1,91 @@
+import json
+import logging
+import os
+import re
+from textwrap import dedent
+
+from openai import AsyncOpenAI
+
+logger = logging.getLogger(__name__)
+
+if not os.getenv("OPENAI_API_KEY"):
+    raise ValueError("OPENAI_API_KEY environment variable is required")
```

Ruff (0.13.1)

Avoid specifying long messages outside the exception class

(TRY003)